encoding.bs

```diff
@@ -732,6 +747,30 @@ part of the ISO 8859 series. In particular, the necessity of the inclusion of <a
 and <a>ISO-8859-16</a> is doubtful for the purpose of supporting existing content, but there are no
 plans to remove these.</p>
 
+<div class=note id=note-latin1-ascii>
+ <p>The <a>windows-1252</a> <a for=/>encoding</a> has various <a for=encoding>labels</a> like
+ "<code>latin1</code>", "<code>iso-8859-1</code>", "<code>ascii</code>", etc. which have
+ historically been confusing for developers. On the web, and in any software that seeks to be
+ web-compatible by implementing the Encoding Standard, these are synonyms: "<code>latin1</code>" and
```

Comment:
I vaguely recall we decided to lowercase standard? Do I misremember?

Reply:
I think it's like the usual proper noun vs. not situation? Encoding Standard, web standards, etc.

Reply:
I found https://github.com/whatwg/html/issues/1156#issuecomment-428346283 on this, but there must be an even older discussion.

Reply:
Well huh, that feels strange to my modern eyes. But happy to align. Will push a fix.

Reply:
Oh. In that comment I was suggesting searching for those strings to find things to fix.

The suggested _replacement_ in that issue is `<cite>Encoding</cite>`.

We could do that (like we do `<cite>Infra</cite>`). It feels a bit strange when a spec talks about itself with a citation, but maybe it's most consistent.

Let me know what you think.

Reply:
How about "this standard"?